prime-angular-frontend/src/app/modules/site-registration/pages/care-setting-page/care-setting-page.component.ts

```diff
@@ -149,6 +152,14 @@ export class CareSettingPageComponent extends AbstractCommunitySiteRegistrationP
     this.patchForm();
   }
 
+  protected submissionRequest(): NoContent {
+    if (this.formState.json.careSettingCode !== CareSettingEnum.DEVICE_PROVIDER) {
+      this.siteFormStateService.deviceProviderFormState.patchValue([]);
```

Comment:
Does this clear the device providers?  Feels like it would need to be `this.siteFormStateService.deviceProviderFormState.patchValue({ individualDeviceProviders: [] });`, or access the getter and invoke clear -`this.siteFormStateService.deviceProviderFormState.individualDeviceProviders.clear()`

Reply:
It works because the patchValue is the method defined in the form state.
![image](https://user-images.githubusercontent.com/59669288/140810726-169a5406-036e-44ab-b587-14f82abbabbf.png)

Reply:
The intent might be clearer if you used `this.siteFormStateService.deviceProviderFormState.clearIndividualDeviceProviders()` then you can reuse it in the `patchValue` method